<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,9">
    <item row="0" column="0" rowspan="2">
     <widget class="QDockWidget" name="dockPlayer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="windowTitle">
       <string>Player</string>
      </property>
      <widget class="QWidget" name="dockContentsPlayer">
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="layoutPlayer"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QDockWidget" name="dockCalculator">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="windowTitle">
       <string>Controller</string>
      </property>
      <widget class="QWidget" name="dockContentsController">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <layout class="QGridLayout" name="layoutCalculator"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuPlayer">
    <property name="title">
     <string>Player</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionPause"/>
    <addaction name="actionSnapshot"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlayer"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="text">
    <string>Snapshot</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../../../Downloads/openshot-qt-develop/openshot-qt-develop/images/openshot - 複製.qrc"/>
 </resources>
 <connections/>
</ui>
